.github/workflows/node.js.yml

```diff
@@ -16,17 +16,17 @@ jobs:
         node-version: [18.x]
         # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
     steps:
-    - uses: actions/checkout@v3
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v3
-      with:
-        node-version: ${{ matrix.node-version }}
-        cache: 'npm'
-    - name: Install dependencies
-      run: npm install
-    - name: Format Check
-      # When fails, please run "npm run format" to your code
-      run: npm run format:check
-    - name: Lint Check
-      # When fails, please run "npm run lint" to your code
-      run: npm run lint:check
+      - uses: actions/checkout@v3
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'npm'
+      - name: Install dependencies
+        run: npm install
+      - name: Lint Check
+        # When fails, please run "npm run lint" to your code
+        run: npm run lint
+      # - name: Format Check
```

Comment:
Format check is breaking the CI. It no longer supported by nx team, see [here](https://github.com/nrwl/nx/issues/4159)